frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.tsx

```diff
@@ -161,23 +161,47 @@ const SortableLayerSettingsContainer = SortableContainer(({ children }: { childr
   return <div>{children}</div>;
 });
 
-const DragHandle = SortableHandle(() => (
-  <div
-    style={{
-      display: "inline-flex",
-      justifyContent: "center",
-      cursor: "grab",
-      alignItems: "center",
-    }}
-  >
-    <MenuOutlined
+type DragHandleProps = {
+  hasLessThanTwoColorLayers: boolean;
+};
+
+function dragHandleIcon(isDisabled: boolean = false) {
+  return (
+    <div
       style={{
-        display: "inline-block",
-        marginRight: 8,
+        display: "inline-flex",
+        justifyContent: "center",
+        cursor: "grab",
+        alignItems: "center",
+        color: isDisabled ? "rgba(0, 0, 0, 0.25)" : "rgba(0, 0, 0, 0.60)",
       }}
-    />
-  </div>
-));
+    >
+      <MenuOutlined
+        style={{
+          display: "inline-block",
+          marginRight: 8,
+        }}
+      />
+    </div>
+  );
+}
+const DragHandle = SortableHandle(({ hasLessThanTwoColorLayers }: DragHandleProps) => {
+  return hasLessThanTwoColorLayers ? (
+    <Tooltip title="Order is only changeable with more than one color layer.">
+      dragHandleIcon(true)
```

Comment:
wouldn't this print the string dragHandleIcon because it's not in {} braces? Why not use something like `<DragHandleIcon isDisabled=... />`?

Reply:
Oh right, I forgot the `{...}` here. The reason why I did not went for the "jsx-version" (<DragHandleIcon isDisabled=... />) is that in this scenario the tooltip does not work. The tooltip somehow needs an immediate child. This problem can be solved by adding a fragment, but this clutters the code imo more than the `{...} ` version. Here is how it would look like with the jsx style:
```javascript
<Tooltip title={`Layer not movable: ${layerType} layers are always rendered on top.`}>
  <>
    <DragHandleIcon isDisabled />
  </>
</Tooltip>
   ```
   As I dislike the  additional fragments, I used the `{ ... } ` style